enterprise/frontend/src/embedding-sdk/types/question.ts

```diff
@@ -20,7 +20,7 @@ export interface SdkQuestionState {
 export interface LoadSdkQuestionParams {
   options?: QueryParams;
   deserializedCard?: Card;
-  questionId?: CardId | null;
+  questionId?: SdkQuestionId | null;
```

Comment:
I modified this to match the current master version.

https://github.com/metabase/metabase/blob/af0188ade10002d948efe3199839cb3f000f164d/enterprise/frontend/src/embedding-sdk/types/question.ts#L40